release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,348 @@
+
+
+[id="deprecated-and-removed-features_{context}"]
+== Deprecated and removed features
+
+Some features available in earlier releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+For the most recent list of major functionality deprecated and removed, see the following table.
+Additional removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+* GA: General Availability
+* TP: Technology Preview
+* DEP: Deprecated
+* REM: Removed
+* NA: Not applicable
+
+.Deprecated and removed features tracker
+[cols="4,1,1,1",options="header"]
+|===
+|Feature |{product-title-short} 4.5 |{product-title-short} 4.6 |{product-title-short} 4.7
+
+|API token authentication for {cloud-redhat-com}^[1]^
+|GA
+|DEP
+|DEP
+
+|`definitions.stackrox.io`
+|DEP
+|DEP
+|DEP
+
+|Google Container Registry integration^[2]^
+|GA
+|DEP
+|DEP
+
+|Kernel support packages and driver download functionality ^[3]^
+|DEP
+|DEP
+|DEP
+
+|Reporting of Istio vulnerabilities
+|DEP
+|DEP
+|DEP
+
+|`rhacs-collector-slim*` images^[4]^
+|DEP
+|DEP
+|DEP
+
+|StackRox Scanner
+|GA
+|DEP
+|DEP
+
+|`/v1/clustercves/suppress` APIs^[5,6]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/clustercves/unsuppress` APIs^[5,6]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/suppress` APIs^[5,6]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/unsuppress` APIs^[5,6]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/summary/counts` endpoint
+|DEP
+|DEP
+|DEP
+
+|Vulnerability Management (1.0) menu item^[7]^
+|DEP
+|DEP
+|DEP
+
+|Vulnerability Report Creator permission
+|DEP
+|DEP
+|DEP
+
+|Scanner V4 support for openSUSE Leap 15.0 and 15.1
+|GA
+|DEP
+|REM
+
+|===
+
+[.small]
+--
+1. API token authentication is deprecated. The corresponding cloud source integration now uses service accounts for authentication.
+
+2. The Google Container Registry integration is deprecated in response to the deprecation of Container Registry. You can use the Artifact Registry as a registry replacement and Scanner V4 as a scanner replacement.
++
+For more information, see link:https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr[Transition from Container Registry] (Google Cloud documentation).
+
+3. Kernel support packages and driver download functionality are deprecated.
+
+4. The `rhacs-collector-slim*` image is deprecated. `rhacs-collector*` image used to contain kernel modules and eBPF probes, but {product-title-short} no longer needs those items. 
++
+The `rhacs-collector*` and the `rhacs-collector-slim*` images are now functionally the same. The `rhacs-collector-slim*` image is planned to be removed in a future release.
```

Comment:
The `rhacs-collector-slim*` images have been removed in 4.7 (they were deprecated in 4.5). Customers will be upgraded to regular `rhacs-collector` images in this release.